test/loadsave.jl

```diff
@@ -46,6 +46,32 @@ func2()
 @test loadmacrotestvar1 == ['a', 'b', 'c']
 @test loadmacrotestvar2 == 1
 
+#Test load_object/save_object function Issue#210
+
+@testset "load_object / save_object" begin
+  @test_throws ArgumentError load_object(fn) #fn already has two objects
+
+  save_object(fn, ['a', 'b', 'c']) #rewrite fn to have one object
+  l1 = jldopen(fn, "r") do f
+    @test length(keys(f)) == 1
+    f["single_stored_object"]
+  end
+  @test l1 == ['a', 'b', 'c']
+
+  #Test with default name
+  l1 = load_object(fn)
+  @test !isdefined(@__MODULE__, :single_stored_object) # should not be in global scope
```

Comment:
Does it harm to have a trivial test? It's not going to consume any significant amount of resources testing this.

Reply:
My problem with this test is not that it is trival, but that it is completely unrelated to the API. 
As I stated below, I think that we can safely assume that no well-behaved function is going to leak things into its surrounding scope, so I don't see why we should test for it.

Reply:
Another question: Even if the variable hypothetically leaked, would this test catch it?
The `@testset` lives in its own scope, no? So, it couldn't even leak to to Module level in the first place?

Reply:
well, in theory the function could leak into any module scope if you explicitely eval something iin that module.
But this just goes back to the point I was trying to make — in Julia, defining names in another scope is a very deliberate act, not something should be tested for because it could happen by accident.

Reply:
Good point. Let's remove that test.